Stripe/STPAPIClient.swift

```diff
@@ -416,45 +410,41 @@ extension STPAPIClient {
         let data = STPMultipartFormDataEncoder.multipartFormData(
             for: [purposePart, imagePart], boundary: boundary)
 
-        var request: NSMutableURLRequest?
-        if let url = URL(string: FileUploadURL) {
-            request = configuredRequest(for: url)
-        }
-        request?.httpMethod = "POST"
-        request?.stp_setMultipartForm(data, boundary: boundary)
-
-        if let request = request {
-            urlSession.stp_performDataTask(
-                with: request as URLRequest,
-                completionHandler: { body, response, error in
-                    var jsonDictionary: [AnyHashable: Any]?
-                    if let body = body {
-                        jsonDictionary =
-                            try? JSONSerialization.jsonObject(with: body, options: [])
-                            as? [AnyHashable: Any]
-                    }
-                    let file = STPFile.decodedObject(fromAPIResponse: jsonDictionary)
-
-                    var returnedError =
-                        NSError.stp_error(fromStripeResponse: jsonDictionary) ?? error
-                    if (file == nil || !(response is HTTPURLResponse)) && returnedError == nil {
-                        returnedError = NSError.stp_genericFailedToParseResponseError()
-                    }
-
-                    if completion == nil {
-                        return
-                    }
-
-                    stpDispatchToMainThreadIfNecessary({
-                        if let returnedError = returnedError {
-                            completion?(nil, returnedError)
-                        } else {
-                            completion?(file, nil)
-                        }
-                    })
+        let url = URL(string: FileUploadURL)!
```

Comment:
hardcoded url should never fail, and if so, we should know right away and not fail silently